specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/locations.json

```diff
@@ -0,0 +1,140 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "HDInsightManagementClient",
+        "description": "The HDInsight Management Client.",
+        "version": "2015-03-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.HDInsight/locations/{location}/usages": {
+            "get": {
+                "tags": [
+                    "Regions"
+                ],
+                "operationId": "Location_ListUsages",
+                "description": "Lists the usages for the specified location.",
+                "x-ms-examples": {
+                    "Get the subscription usages for specific location": {
+                        "$ref": "./examples/GetHDInsightUsages.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK response definition.",
+                        "schema": {
+                            "$ref": "#/definitions/UsagesResult"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "LocalizedName": {

```

Comment:
In the objects defined inside definition for this swagger there is not a single property that is sent to server so please mark every property as `"readOnly": true`

Reply:
Sounds good.